tron/bin/action_runner.py

```diff
@@ -93,6 +93,24 @@ def build_environment(run_id, original_env=None):
     return new_env
 
 
+def build_labels(run_id, original_labels=None):
+    if original_labels is None:
+        original_labels = dict()
+
+    try:
+        run_num = run_id.split(".", maxsplit=3)[2]
```

Comment:
small suggestion:
```suggestion
        # reminder: the format here is "namespace.job.run_num.action"
        _, _, run_num, _ = run_id.split(".", maxsplit=3)
```
would allow us to drop the `[2]` and add a little documentation for future yelpers readin this function :)

Reply:
added